containers/ecr-viewer/src/app/services/formatService.ts

```diff
@@ -424,6 +425,15 @@ export const formatAge = (age: Age | undefined): string | undefined => {
   return `${getFormattedMonths(totalMonths)}${days} day${makePlural(days)}`;
 };
 
+/**
+ * Returns the value of a Reference
+ * @param reference the reference being formatted
+ * @returns .reference value of the supplied reference
+ */
+export const formatReference = (reference: Reference): string => {
+  return reference?.reference || "";
+};
```

Comment:
```suggestion
export const formatReference = (reference: Reference | undefined): string | undefined => {
  if (!reference) return;

  return reference.reference;
};
```

This means we'll need the `|| ""` at the call site for `evaluateValue`, but it's more consistent with how most of these formatters work. Can you also move this to above `formatAge` - having this sandwiched between it and its helper below is a bit odd in terms of file flow

Reply:
Swapped this around as you mentioned, let me know how it looks